camera_calibration/src/camera_calibration/calibrator.py

```diff
@@ -660,7 +660,10 @@ def set_alpha(self, a):
         # NOTE: Prior to Electric, this code was broken such that we never actually saved the new
         # camera matrix. In effect, this enforced P = [K|0] for monocular cameras.
         # TODO: Verify that OpenCV #1199 gets applied (improved GetOptimalNewCameraMatrix)
-        ncm, _ = cv2.getOptimalNewCameraMatrix(self.intrinsics, self.distortion, self.size, a)
+        center_principle = (self.calib_flags & cv2.CALIB_FIX_PRINCIPAL_POINT) > 0
```

Comment:
isnt `cv2.CALIB_FIX_PRINCIPAL_POINT` a constant? What does adding this comparison do?

Reply:
`center_principle` is a boolean that indicates if `self.calib_flags` contains the `cv2.CALIB_FIX_PRINCIPAL_POINT` flag

Reply:
I read this as `calib_flags and cv2.CALIB_...`, I'm not sure checking a `cv2` const makes sense - there's nothing here that compares if calib_flags has this cv2 constant contained with in it that I know of.